tamr_toolbox/workflow/jobs.py

```diff
@@ -188,3 +189,29 @@ def get_upstream_projects(project: Project) -> List[Project]:
     upstream_projects = _find_upstream_projects(project)
 
     return upstream_projects
+
+
+def get_enrichment_project_output_dataset(project: Project) -> str:
+    """Check for the output dataset for a specified enrichment project
+
+    Args:
+        project: the tamr enrichment project for which associated output dataset name is retrieved
+    """
+    if project.type == "ENRICHMENT":
```

Comment:
Rather than returning a string for ENRICHMENT projects and returning None for other project types, you should raise an error if a non-enrichment project is passed:
```suggestion
    if project.type != "ENRICHMENT":
        raise ValueError(f"{project.name} is not an enrichment project")
```